device/mellanox/x86_64-mlnx_msn4800_simx-r0/ACS-MSN4800/pg_profile_lookup.ini

```diff
@@ -0,0 +1,23 @@
+#  PG lossless profiles.
+# speed cable  size   xon    xoff   threshold
+  10000    5m 52224  19456  32768          0
+  25000    5m 52224  19456  32768          0
+  40000    5m 53248  19456  33792          0
+  50000    5m 53248  19456  33792          0
+ 100000    5m 53248  19456  33792          0
+ 200000    5m 55296  19456  35840          0
+ 400000    5m 86016  37888  38912          0
+  10000   40m 53248  19456  33792          0
+  25000   40m 55296  19456  35840          0
+  40000   40m 57344  19456  37888          0
+  50000   40m 58368  19456  38912          0
+ 100000   40m 63488  19456  44032          0
+ 200000   40m 74752  19456  55296          0
+ 400000   40m 124928 37888  77824          0
+  10000  300m 60416  19456  40960          0
+  25000  300m 73728  19456  54272          0
+  40000  300m 86016  19456  66560          0
+  50000  300m 95232  19456  75776          0
+ 100000  300m 137216 19456  117760         0
+ 200000  300m 223232 19456  203776         0
+ 400000  300m 420864 37888  373760         0
```

Comment:
The headroom depends on the cable length, speed, MTU, and gearbox delay. 
Currently, the headroom is indexed by (speed, cable length) tuple because all the ports share the same MTU and gearbox model and delay.
But on 4800 the thing is different. It is possible that different gearbox models are equipped on different line cards, which means two ports from different line cards can have different gearbox models and then delays, which in turn means the (speed, cable length) tuple is no longer enough to index the headroom.
This can be handled by dynamic buffer calculation but not by the traditional buffer manager. 
I'm not sure how this should be handled if the traditional buffer manager is used.
Need to discuss with Arch.